tests/test_dfe.py

```diff
@@ -8,14 +8,13 @@
 
 def test_exhaustive_state_dfe():
     texpt = exhaustive_state_dfe(program=Program(X(0), X(1)), qubits=[0, 1])
-    assert len(texpt) == 3 ** 2 - 1
+    assert len(texpt) == 2 ** 2 - 1
 
 
 def test_exhaustive_dfe():
     texpt = exhaustive_process_dfe(program=Program(Z(0)), qubits=[0])
     assert len(texpt) == 7 ** 1 - 1
 
-
```

Comment:
We want two empty lines between top-level functions, as is the PEP standard.

Reply:
Should be fixed now